### PR TITLE
Option for multiple bounding and ignore boxes

### PR DESCRIPTION
**Pull request by ashwinmenkudle**

### CODE REVIEW

**Comment by wswebcreation** [comment]
Tnx for this PR, I was also looking into this. Saves me some time ;-)

File: resemble.js [hunk 1]
```diff
@@ -107,8 +107,8 @@ var isNode = new Function("try {return this===global;}catch(e){return false;}");
 
         var errorPixel = errorPixelTransform.flat;
         var errorType;
-        var boundingBox;
-        var ignoredBox;
+        var boundingBoxs;
```
Comment (jamescryer) [inline resemble.js]: Please change to `boundingBoxes` and `ignoredBoxes`

File: resemble.js [hunk 2]
```diff
@@ -148,22 +148,40 @@ var isNode = new Function("try {return this===global;}catch(e){return false;}");
...
+            			break;
+            		}
+            	}
             }
-
+            
+            if(selected === undefined && ignored === undefined) {
+            	return true;
+            }
+            if(selected === false && ignored === true) {
```
Comment (jamescryer) [inline resemble.js]: Could we remove these last three if statements and simply `return selected && ignored`

**Comment by wswebcreation** [comment]
What's the status on the PR?
I'd like to use the module instead of using a fork

**Comment by chrischjh** [comment]
Also waiting for this

**Comment by jamescryer** [comment]
Changes are now merged with API changes and new tests

**Pull request merged**